### DISCUSSION TITLE
Domain name not stable

### DISCUSSION THREAD

**Discussion by rickycheuk** [vercel/community#3738] [Help]
### Summary

I bought a domain on GoDaddy and configured the nameserver to Vercel, setup everything but the domain name is very unstable, sometimes it works, sometimes says site cannot be reached with error DNS_PROBE_FINISHED_NXDOMAIN. 

I have tried with different devices and networks but errors are the same.

### Steps to Reproduce

- buy domain from GoDaddy
- add domain name to Vercel
- change nameservers of of domain to Vercel's nameservers
- wait for DNS to propagate
- the domain name works sometimes but says unable to reach most of the time

**Comment by amyegan** [comment]
I can look for more information about this error if you share the domain here
